ci-operator/step-registry/quay-operator/deploy-aws-s3/quay-operator-deploy-aws-s3-commands.sh

```
@@ -0,0 +1,294 @@
+    managed: true
+  - kind: route
+    managed: true
+EOF
+
+for _ in {1..60}; do
+  if [[ "$(oc -n quay-enterprise get quayregistry quay -o jsonpath='{.status.conditions[?(@.type=="Available")].status}' || true)" == "True" ]]; then
+    echo "Quay is in ready status" >&2
+    oc -n quay-enterprise get quayregistries -o yaml >"$ARTIFACT_DIR/quayregistries.yaml"
+    oc get quayregistry quay -n quay-enterprise -o jsonpath='{.status.registryEndpoint}' > "$SHARED_DIR"/quayroute || true
+    quay_route=$(oc get quayregistry quay -n quay-enterprise -o jsonpath='{.status.registryEndpoint}') || true
+    curl -k -X POST $quay_route/api/v1/user/initialize --header 'Content-Type: application/json' \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)